src/components/about-us/layout.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ABOUT_CLASSES,
+  ABOUT_TEAM,
+  DEVELOPERS,
+  LABEL_RSS_LINK,
+  RSS_LINK,
+  TEAM_SLOGAN,
+} from '../../pages/about-us/constants';
+import { LAYOUT_CLASSES } from '../../pages/product/constants';
+import { createDiv, createH2, createP } from '../../utils/create-elements/create-tags';
+import { createDeveloperCart } from './developer-cart';
+import { createLinkWithPicture } from './link-with-picture';
+
+export function AboutUsLayout(): HTMLDivElement {
+  const cardsContainer = createDiv({
+    classes: ABOUT_CLASSES.cardsContainer,
+  });
+
+  DEVELOPERS.map((DeveloperInfo) => cardsContainer.append(createDeveloperCart(DeveloperInfo)));
```

Comment:
it's better to use a lowercase letter for the argument `DeveloperInfo`, 
```suggestion
  DEVELOPERS.map((DeveloperInfo) => cardsContainer.append(createDeveloperCart(developerInfo)));
```

Reply:
I'm not sure whether it's a good practice to use map method here, I now that eslint forbids us to use forEach method, we could use `for of` loop here

Reply:
done